### DISCUSSION TITLE
Yi Outdoor Camera 1080p 9CUS

### DISCUSSION THREAD

**Discussion by STnmsk** [alienatedsec/yi-hack-v5#266] [Q&A]
Hello!
I have Yi Outdoor Camera 1080p, Serial number - 9CUS, Firmware - 3.0.0*
Can I install this hack? If not, did yor plan to add this camera in your hack?

**Comment by alienatedsec** [comment]
Yes. Use the latest pre-release. But you need to follow the guideline and extract files and the folder to the SD card.

**Reply by STnmsk** [reply]
I did it. I uploaded the necessary files from the archive to the SD card and inserted into the camera. After turning on the camera, it worked, but its web interface is still not available.

**Reply by alienatedsec** [reply]
The port 8080 is the web interface

**Reply by STnmsk** [reply]
I know. But it not work.

**Reply by STnmsk** [reply]
Have you any idea, how to fix it?

**Reply by alienatedsec** [reply]
Please follow the guideline to flash the baseline firmware. Not sure what else I can advise.
